EIPS/eip-7870.md

```diff
@@ -0,0 +1,145 @@
+
+### Bandwidth
+
+#### Local Block Builders
+
+- **Recommended**: 100 Mbps download, 50 Mbps upload.
+- **Rationale**:  
+  - Distributing blocks is highly bandwidth-intensive.  
+  - If a builder cannot meet these speeds, they risk slower propagation and causing late blocks.  
+  - In extreme cases, a low-bandwidth node could propose partially full blocks or one that includes fewer blobs to mitigate slow broadcast.
+
+#### Attesters
+
+- **Recommended**: 50 Mbps download, 25 Mbps upload.
+- **Minimum tested**: 15 Mbps (as per [ethPandaOps](https://ethpandaops.io/) simulations where 40% of the network had Gigabit connections and the other 60% had 15 Mbps upload).
+  - However, real-world performance depends on peer network topology. A node with poor bandwidth could in theory quickly share data with a well-connected peer with good bandwidth which means that this peer could quickly seed the network.
+- **Rationale**:
+  - 25 Mbps was chosen as a buffer to account for these miscellaneous factors that are harder to predict.
+
+#### Validators Using MEV-Boost
+
+- **Recommended**: 50 Mbps download / 25 Mbps upload.
+- **Rationale**:
+  - Most MEV-relay interactions involve fetching bundles and block headers, which can be done within typical broadband speeds.
+  - While the local validator will also share the block with its peers, the relay will do the same which reduces the need for local bandwidth.
+  - However, there may be cases where your validator will still build a local block, such as if no MEV-relay responds or if the value of the MEV reward is lower than the minimum configuration set in MEV-Boost. In these circumstances, the recommendations for **Local Block Builders** would be relevant.
+
+#### Full Nodes
+
+- **Recommended**: 50 Mbps download / 15 Mbps upload.
+- **Rationale**:
+  - Full nodes currently participate in sampling and must track the chain tip, but are not as latency-sensitive as attesters or local block builders.
+  - They can operate on lower bandwidth but risk being a slot or more behind the chain if bandwidth capacity is severely limited.
+
+## Quick Reference Summary
+
+- **Full Node**  
+  - **Storage**: 4 TB NVMe  
+  - **RAM**: 32 GB  
+  - **CPU**: 4 cores / 8 threads (~1000 ST / ~3000 MT PassMark)  
+  - **Bandwidth**: 50 Mbps down / 15 Mbps up  
+
+- **Attester**  
+  - **Storage**: 4 TB NVMe  
+  - **RAM**: 64 GB  
+  - **CPU**: 8 cores / 16 threads (~3500 ST / ~25000 MT PassMark)  
+  - **Bandwidth**: 50 Mbps down / 25 Mbps up  
+
+- **Local Block Builder**  
+  - **Storage**: 4 TB NVMe  
+  - **RAM**: 64 GB  
+  - **CPU**: 8 cores / 16 threads (~3500 ST / ~25000 MT PassMark)  
+  - **Bandwidth**: 100 Mbps down / 50 Mbps up  
+
+## Backwards Compatibility
+
+This EIP is informational and requires no protocol changes. We recommend that future EIPs include an assessment of their impact on these hardware recommendations.
+
+## Security Considerations
```

Comment:
One thing I'd suggest doing prior to applying that is to do any research among community either by pool on X or anything else which can have bigger outreach.

If it will appear that 51% of users have weaker than specified setup (which is super unlikely but still) we could potentially open a route for some unexpected attacks because we will test on too good hardware.

At least worth to know in case of noticed issue which affect weaker setups how much network (in terms of nodes) is put in danger.

Reply:
We could alternatively, test against the old NUCs for a year too though the current idea was that instead of us polling N validators, we would give them a tool to allow them to assess themselves and decide when they would want to update